tests/unit/core/exporter/exportable_code/conftest.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import importlib
+import importlib.util
+import sys
+from pathlib import Path
+
+from otx.core.exporter.exportable_code import demo
+
+# exportable_code is standalone package and files in exportable_code import 'demo_package' which is same as
+# 'otx.core.exporter.exportable_code.demo.demo_package'. To avoid import error while running test, need to
+# import 'otx.core.exporter.exportable_code.demo.demo_package' and register it as 'demo_package'.
+demo_package_file = Path(demo.__file__).parent / "demo_package" / "__init__.py"
+spec = importlib.util.spec_from_file_location("demo_package", demo_package_file)
+module = importlib.util.module_from_spec(spec)
+sys.modules["demo_package"] = module
```

Comment:
Would you attach an error log if we don't have this modification as comments here as well?

Reply:
In addition, this should be a fixture to make it restrictive for the tests under `tests/unit/core/exporter/exportable_code`

```python
@pytest.fixture(scope="module")
def fxt_fix_exportable_code_import_error():
    # exportable_code is standalone package and files in exportable_code import 'demo_package' which is same as
    # 'otx.core.exporter.exportable_code.demo.demo_package'. To avoid import error while running test, need to
    # import 'otx.core.exporter.exportable_code.demo.demo_package' and register it as 'demo_package'.
    demo_package_file = Path(demo.__file__).parent / "demo_package" / "__init__.py"
    spec = importlib.util.spec_from_file_location("demo_package", demo_package_file)
    module = importlib.util.module_from_spec(spec)

    tmp_mod_demo_package = sys.modules.get("demo_package", None)
    sys.modules["demo_package"] = module

    yield

    # Revert sys.modules["demo_package"] modification
    if tmp_mod_demo_package:
        sys.modules["demo_package"] = tmp_mod_demo_package
    else:
        sys.modules.pop("demo_package")
```

Reply:
Thanks for detailed explanation. I also agree with your opinion but it seems that fixture is executed after import, which makes error.
![image](https://github.com/openvinotoolkit/training_extensions/assets/92974184/f1e32e69-15ca-40e7-ac4f-d0c602f45389)

Reply:
> do you mean attaching error log at comment?

Yes

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3212/commits/cfc87b25f3ad44dfd07350cc8c7727c79a6eb298